engine/src/flutter/lib/ui/painting/image_decoder_unittests.cc

```diff
@@ -1007,88 +1010,6 @@ TEST_F(ImageDecoderFixtureTest,
   PostTaskSync(runners.GetIOTaskRunner(), [&]() { io_manager.reset(); });
 }
 
-TEST_F(ImageDecoderFixtureTest, MultiFrameCodecDidAccessGpuDisabledSyncSwitch) {
```

Comment:
We don't actually use the GPU sync switch in the multiframe codec. Instead iOS uses host visible that is always updated ignoring the switch.